fa/deep-learning-tips-and-tricks.md

```diff
@@ -0,0 +1,585 @@
+
+<div dir="rtl">
+نرخ‌های یادگیری سازگارشونده ― داشتن نرخ یادگیری متغیر در فرآیند آموزش یک مدل،  می‌تواند زمان آموزش را کاهش دهد و راه‌حل بهینه عددی را بهبود ببخشد. با آنکه بهینه ساز Adam محبوب‌ترین متد مورد استفاده است، دیگر متدها نیز می‌توانند مفید باشند. این متد ها در جدول زیر به اختصار آمده‌اند:
+</div>
+
+<br>
+
+
+**41. [Method, Explanation, Update of w, Update of b]**
+
+<div dir="rtl">
+[روش، توضیح، بروزرسانی w، بروزرسانی  b]
+</div>
+
+<br>
+
+
+**42. [Momentum, Dampens oscillations, Improvement to SGD, 2 parameters to tune]**
+
+<div dir="rtl">
+[Momentum، نوسانات را تعدیل می‌دهد، بهبود SGD، 2 پارامتر نیاز به تنظیم دارند]
+</div>
+
+<br>
+
+
+**43. [RMSprop, Root Mean Square propagation, Speeds up learning algorithm by controlling oscillations]**
+
+<div dir="rtl">
+[RMSprop، انتشار جذر میانگین مربعات، سرعت بخشیدن به الگوریتم یادگیری با کنترل نوسانات]
+</div>
+
+<br>
+
+
+**44. [Adam, Adaptive Moment estimation, Most popular method, 4 parameters to tune]**
+
+<div dir="rtl">
+[Adam، تخمین سازگارشونده گشتاور، محبوب‌ترین متد، 4 پارامتر نیاز به تنظیم دارند]
+</div>
+
+<br>
+
+
+**45. Remark: other methods include Adadelta, Adagrad and SGD.**
+
+<div dir="rtl">
+نکته: سایر متدها  شامل Adadelta، Adagrad و SGD هستند.
+</div>
+
+<br>
+
+
+**46. Regularization**
+
+<div dir="rtl">
+نظام‌بخشی
+</div>
+
+<br>
+
+
+**47. Dropout ― Dropout is a technique used in neural networks to prevent overfitting the training data by dropping out neurons with probability p>0. It forces the model to avoid relying too much on particular sets of features.**
+
+<div dir="rtl">
+برون‌اندازی – برون‌اندازی روشی است که در شبکه‌های عصبی برای جلوگیری از برارزش شدن بر روی داده‌های آموزشی با حذف تصادفی نورون‌ها با احتمال p>0 استفاده می‌شود. این روش مدل را مجبور می‌کند تا از تکیه کردن بیش از حد بر روی مجموعه خاصی از ویژگی‌ها خودداری کند.
+</div>
+
+<br>
+
+
+**48. Remark: most deep learning frameworks parametrize dropout through the 'keep' parameter 1−p.**
+
+<div dir="rtl">
+نکته: بیشتر فریم ورک‌های یادگیری عمیق برون‌اندازی را به شکل پارامتر ‘keep’  1-p درمی-آورند.
+</div>
+
+<br>
+
+
+**49. Weight regularization ― In order to make sure that the weights are not too large and that the model is not overfitting the training set, regularization techniques are usually performed on the model weights. The main ones are summed up in the table below:**
+
+<div dir="rtl">
+نظام‌بخشی وزن – برای اطمینان از اینکه (مقادیر) وزن‌ها زیادی بزرگ نیستند و مدل به مجموعه آموزش بیش‌برارزش نیست، روشهای نظام‌بخشی معمولا بر روی وزن‌های مدل اجرا می‌شوند. اصلی‌ترین آنها در جدول زیر به اختصار آمده‌اند:
+</div>
+
+<br>
+
+
+**50. [LASSO, Ridge, Elastic Net]**
+
+<div dir="rtl">
+[LASSO, Ridge, Elastic Net]
+</div>
+<br>
+
+**50 bis. Shrinks coefficients to 0, Good for variable selection, Makes coefficients smaller, Tradeoff between variable selection and small coefficients]**
```

Comment:
"bis" ?!

Reply:
It was not included in the other translation.